src/agl/WebRuntimeAGL.h

```diff
@@ -40,10 +41,22 @@ class Launcher {
   std::vector<pid_t> m_pid_v;
 };
 
+class TinyProxy {
+ public:
+  int port() { return port_; }
+  void setPort(int port) { port_ = port; }
+
+  TinyProxy();
+private:
+  int port_;
+};
+
 class SharedBrowserProcessWebAppLauncher : public Launcher {
 public:
   int launch(const std::string& id, const std::string& uri) override;
   int loop(int argc, const char** argv, volatile sig_atomic_t& e_flag) override;
+private:
+  std::unique_ptr<TinyProxy> tiny_proxy;
```

Comment:
We could consider using the Qt coding standard as widely adopted in WAM, or move already to the one we expect in the future (Chromium coding style).

I'd be happier with newer chromium style as we expect to also do QtLess, so it'd be tiny_proxy_